runner/src/server/plugins/engine/models/viewModel.ts

```diff
@@ -399,10 +399,19 @@ function Item(
     componentValue = null;
   }
 
+  let componentLabel = component.localisedString(component.title);
+
+  if (
+    component.options.required == false &&
+    !componentLabel.includes("optional")
+  ) {
+    componentLabel += " (optional)";
+  }
```

Comment:
Hmm - could we use a truthy value on `required` as in a lot of components it's optional? (not sure the context of which component is being used here) 

Also, instead of doing a string search for "optional", could we do a more explicit check for "(optional)" happening at the end?  I'm just worried by coincidence "optional" may be part of the label but not at the end, and as some other context.

Something like:

> 1. What is the name of the primary contact for this funding request, and the optional secondary contact? 
> 2. Does your project collaborate with any other organizations or entities, and if so, are any of them part of the "optional partners" list we provided?
> 3. If this funding request is not fully approved, will the project still proceed with a reduced scope or optional activities?

Just some random examples to show what I mean.

i.e
```suggestion
  if (
    !component.options.required &&
    !componentLabel.toLowerCase().endsWith("(optional)")
  ) {
    componentLabel += " (optional)";
  }
```

Reply:
Ah, does no required field mean it is required then? 🤔